### ISSUE TITLE
No @media queries to overwrite in appleWebKit if changed `justify-content` to centre or left

### ISSUE THREAD

**Issue by hashmil** [elementor/elementor#13897]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

I have encountered this bug only with Safari and iOS browsers (Chrome & Safari or Apple WebKit browsers).
The Flex alignments for the Share buttons components are not working as intended.

It's in the `frontend.min.css` file of Elementor. There is a selector for appleWebkit for the align-right situation but no @media queries to overwrite it if changed to centre or left:

Line 3987 on `frontend.css` in `/assets/css/`
```
.e--ua-appleWebkit .elementor-share-buttons--align-right, .e--ua-appleWebkit .elementor-widget-social-icons.e-grid-align-right {
--justify-content: var(--flex-right,flex-end);
}
```

**image_01.jpg** is in chrome on my mac, where I've set it to centre align on the mobile breakpoint.
![image_01](https://user-images.githubusercontent.com/4124458/107859833-28fdab80-6e34-11eb-9ab3-89ae87a19bec.jpg)

**image_02** is on Safari, and you can see it's right-aligned and not working. When I've re-written the code (which has been commented out for the screenshot) then it works.
![image_02](https://user-images.githubusercontent.com/4124458/107859835-2ac76f00-6e34-11eb-8838-e785f7ec29e4.jpg)

Basically, I've added the following custom CSS for it to work:
![image_03](https://user-images.githubusercontent.com/4124458/107859870-56e2f000-6e34-11eb-906a-354df3f4320c.jpg)

`.e--ua-appleWebkit .elementor-share-buttons--align-right, .e--ua-appleWebkit .elementor-widget-social-icons.e-grid-align-right {
    --justify-content: center;
}`

## Steps to reproduce

1. add the Share Buttons (or the Social Icons) element.
2. in the desktop breakpoint under the Content tab set the alignment to right
3. in the mobile breakpoint under the COntent tab set the alignment to center
4. open safari - resize to mobile size (might have open inspect element to scale down the browser window that small), or the browser in iOS and see the results.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
https://pastebin.com/zptCAjsZ
</details>

**Comment by askwpgirl** [comment]
I can reproduce this bug as well. It appears to only effect the widget when it is set to align right for Desktop but a different alignment for tablet or mobile, due to this one align-right selector for appleWebkit in the frontend.css. If the social widget is originally set to center or align left, this isn't an issue. I'm not sure why this and only this selector is in frontend.css. It looks like it could be removed or needs to be modified to account for different alignment options for tablet and mobile.

**Comment by DorShahar** [comment]
Merged issue

I merged this issue with  #13122 , which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!
